src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -437,3 +438,8 @@ void SensorFusion::resetWorldComponents()
     enemy_team_filter    = RobotTeamFilter();
     possession           = TeamPossession::FRIENDLY_TEAM;
 }
+
+void SensorFusion::setVirtualObstacles(TbotsProto::VirtualObstacles &virtual_obstacles)
```

Comment:
should be marked `const` so that the class takes copy. I believe with this current definition, we are at risk of a dangling pointer to a locally defined variable